extension/src/cli/dvc/executor.ts

```diff
@@ -71,6 +72,10 @@ export class DvcExecutor extends DvcCli {
     return this.blockAndExecuteProcess(cwd, Command.COMMIT, ...args, Flag.FORCE)
   }
 
+  public config(cwd: string, ...args: Args) {
```

Comment:
[F] Config doesn't really fit with the `reader`/`executor` model as it can both read and write the config. For now, I've put it into `executor`.